leapp/reporting/__init__.py

```diff
@@ -378,12 +381,22 @@ def _create_report_object(entries):
     _sanitize_entries(entries)
     for entry in entries:
         entry.apply(report)
+
     if Groups.INHIBITOR in report.get('groups', []):
+        if Groups.ERROR in report.get('groups', []):
+            # *error != inhibitor*
+            msg = ('Only one of Groups.ERROR and Groups.INHIBITOR can be set at once.'
+                   ' Maybe you were looking for Groups.FAILURE?')
+            raise ValueError(msg)
+
         # Before removing Flags, the original inhibitor detection worked
         # by checking the `flags` field; keep the `flags` field until we drop Flags completely
         # Currently we know that 'flags' does not exist otherwise so it's
         # safe to just set it
         report['flags'] = [Groups.INHIBITOR]
+    if Groups.ERROR in report.get('groups', []):
+        # see above for explanation
+        report['flags'] = [Groups.ERROR]
```

Comment:
also, we could keep error as a tag, but it does not fit there well. let's discuss that in the morning

Reply:
thinking about that after the sleep, I vote to go the most safe way. We will drop it from the json file completely and it will be covered properly the next time.

Reply:
In this case it shows up as in the 1.2.0 schema:
```
"groups": [
    "error"
],
```
And not flags/tags field.
This should be the problematic line:
https://github.com/oamg/leapp/pull/840/files#diff-a0dcff5a59a744244ee3056a16cd7fca4bbcc6cd573c6d6e9dd26ddb4205723eL151